collect_app/src/main/java/org/odk/collect/android/dao/FormsDao.java

```diff
@@ -44,6 +44,14 @@ public Cursor getFormsCursor(String selection, String[] selectionArgs) {
         return getFormsCursor(null, selection, selectionArgs, null);
     }
 
+    public int deleteFormsForFormId(String[] selectionArgs) {
```

Comment:
There has a been a few tickets related to limitation on sqlite number of params in a statement (999) is the hard limit. So I'm not sure whether we should introduce slicing in this delete operation or not 👍